### PR TITLE
Dev -> Prod v10.14.0

### PR DESCRIPTION
**Pull request by jkonrath-postman**
TW-1087 Workspace templates #4784
TW-1090 RFA viewer elevate workspace role #4786
TW-879 Related collections and flows section #4762
TW-872 Protocol selector UI changes #4788
TW-1093 Postman VS Code extension #4780
TW-1089 Flows editor enhancements #4764
TW-1096 Doc enable video embeds in documentation #4781
TW-1091 Doc Golang SDK with Gin framework support for Live Collections and
TW-1002 Live Collections conflict resolution #4774
TW-1107 CNX changes #4782
TW-693 Doc GraphQL #4607
TW-1100 Flows minimap and grouping #4785
TW-1116 Update disable sync version to include lightweight API client #4793
TW-1104 Account vs Team #4787
Remove frontmatter to get under search limit #4795

### CODE REVIEW

File: src/pages/docs/collections/live-collections/live-collections-setup.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ contextual_links:
...
+1. To create a new Live Collection, select a collection, select the Live Collections icon <img alt="Live Collections icon" src="https://assets.postman.com/postman-docs/v10/icon-live-collections.jpg#icon" width="16px"> in the right sidebar, then select **Enable Live**.
 
-1. Select your application language: **Node.js** or **Python**.
+1. Select your application language: **Node.js**, **Python**, or **Go**.
 
-1. Select a framework. For Node.js, select **Express.js**, **Sails.js**, or **Fastify**. For Python, select **Flask**.
+1. Select a framework. For Node.js, select **Express.js**, **Sails.js**, or **Fastify**. For Python, select **Flask**. For Go, select **Gin**.
 
-1. Install the Postman SDK in the application’s root directory. You're provided with a installation command that uses `npm` or `pip`. Copy the command and enter it on your server to install the SDK.
+1. Install the Postman SDK in the application’s root directory. You're provided with a installation command that uses `npm`, `pip`, or `go get`. Copy the command and enter it on your server to install the SDK.
```
Comment (petewitcher-postman) [inline src/pages/docs/collections/live-collections/live-collections-setup.md:22]:
```suggestion
1. Install the Postman SDK in the application’s root directory. You're provided with an installation command that uses `npm`, `pip`, or `go get`. Copy the command and enter it on your server to install the SDK.
```

File: src/pages/docs/sending-requests/graphql/graphql-http.md [hunk 2]
```diff
@@ -1,8 +1,7 @@
 ---
-title: "Querying with GraphQL"
-order: 37
-page_id: "graphql"
-updated: 2022-02-22
+title: "Using GraphQL with the HTTP request interface"
+page_id: "graphql-http"
+updated: 2023-02-24
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/graphql/graphql-http.md:4]:
```suggestion
updated: 2023-05-15
```

File: src/pages/docs/sending-requests/websocket/websocket.md [hunk 3]
```diff
@@ -52,11 +51,10 @@ In addition to raw WebSocket connections, Postman also supports Socket.IO connec
 
 You can create a WebSocket request from the sidebar in Postman.
 
-1. Select  **New > WebSocket Request** to open a new tab. (In the Postman desktop app, you can also select **⌘+N** or **Ctrl+N**.)
+1. Select **New > WebSocket** to open a raw WebSocket request in new tab. You can also select **New > Socket.IO** to open a Socket.IO request in new tab. (In the Postman desktop app, you can also select **⌘+N** or **Ctrl+N**.)
```
Comment (petewitcher-postman) [inline src/pages/docs/sending-requests/websocket/websocket.md:54]:
```suggestion
1. Select **New > WebSocket** to open a raw WebSocket request in a new tab. You can also select **New > Socket.IO** to open a Socket.IO request in a new tab. (In the Postman desktop app, you can also select **⌘+N** or **Ctrl+N**.)
```

**Review by petewitcher-postman — APPROVED** [review]
LGTM! Added a minor comment and fixed a few typos.

**Review by MatthewPinsker-postman — APPROVED** [review]
LGTM!

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/collections/live-collections/update-live-collections.md [hunk 4]
```diff
@@ -35,6 +35,10 @@ To manually update a Live Collection, do the following:
 
     <img alt="Live Collections review changes" src="https://assets.postman.com/postman-docs/v10/live-collections-review-changes.jpg" />
 
+1. If you made changes in your Live Collection and the same component is changed by incoming changes from the service, a conflict will occur. These conflicts are shown when you review incoming changes. You can select if you want to keep user updates or service updates for each part of the conflicted requests.
```
[the remark this answers is not in the document]
Comment (jkonrath-postman) [reply src/pages/docs/collections/live-collections/update-live-collections.md:38]: No, but it is a weird wording.

**Pull request merged**